src/components/theme-switch/user-dropdown-switcher/user-dropdown-switcher.module.css

```diff
@@ -0,0 +1,17 @@
+.label {
+	composes: hds-typography-body-200 from global;
+	composes: hds-font-weight-medium from global;
+}
+
+.listItem {
+	display: flex;
+	border-radius: 5px;
+	color: var(--token-color-foreground-primary);
+	grid-gap: 8px;
+	gap: 8px;
+	width: 100%;
+	border: none;
+	padding: 8px 10px 8px 16px;
+	justify-content: space-between;
+	align-items: center;
+}
```

Comment:
🎨 `justify-content: space-between` will override `gap`/`grid-gap` if there's more space between than the value for those gaps, so both aren't needed here! Based on spacing I'm seeing in Figma ([link](https://www.figma.com/file/v0Hyqvjdf6QJAvRkdIBV24/Dark-Mode?node-id=1166-44816&t=gB7xuXM6NrfgBgE9-11)), I'd probably define `gap: 12px` and remove `justify-content`. This would have to be done in combination with some other tweaks I noted like the `max-width` comment above.

![CleanShot 2023-03-31 at 13 00 05](https://user-images.githubusercontent.com/43934258/229183861-234ab044-1380-4c7c-a672-04dc1c904f9b.png)